tests/dialects/test_bigquery.py

```diff
@@ -218,6 +218,36 @@ def test_bigquery(self):
             "SELECT * FROM UNNEST(x) WITH OFFSET AS offset EXCEPT DISTINCT SELECT * FROM UNNEST(y) WITH OFFSET AS offset",
         )
 
+        self.validate_all(
+            "SELECT LAST_DAY(CAST('2008-11-25' AS DATE), MONTH)",
+            read={
+                "snowflake": "SELECT LAST_DAY(CAST('2008-11-25' AS DATE), MONTH)",
```

Comment:
snowflake allows abbrevations, the work that i recently did https://docs.snowflake.com/en/sql-reference/functions-date-time

you need to use the abbrevation mapping to support last_day(..., mons) as an example

Reply:
ah I see, thanks